integrationtests/src/test/java/io/kroxylicious/proxy/filter/OutOfBandSendFilter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter;
+
+import java.util.stream.Collectors;
+
+import org.apache.kafka.common.message.CreateTopicsRequestData;
+import org.apache.kafka.common.message.CreateTopicsResponseData;
+import org.apache.kafka.common.message.DescribeClusterRequestData;
+import org.apache.kafka.common.message.DescribeClusterResponseData;
+import org.apache.kafka.common.message.RequestHeaderData;
+import org.apache.kafka.common.protocol.ApiKeys;
+import org.apache.kafka.common.protocol.Errors;
+
+public class OutOfBandSendFilter implements DescribeClusterRequestFilter {
+    @Override
+    public void onDescribeClusterRequest(short apiVersion, RequestHeaderData header, DescribeClusterRequestData request, KrpcFilterContext context) {
+        CreateTopicsRequestData request1 = new CreateTopicsRequestData();
```

Comment:
OutOfBandSendingFilter could accept the name of the class to send.   We don't need it now, but that would leave open the possibility for this class to accept a payload (as JSON).